resources/ctia-default.properties

```diff
@@ -17,6 +17,9 @@ ctia.http.port=3000
 ctia.http.access-control-allow-origin=http://(localhost|127.0.0.1)(:\\d+)?
 ctia.http.access-control-allow-methods=get,post,put,patch,delete
 
+ctia.http.events.timeline.max-seconds=5
+ctia.http.bundle.max-relationships=1000
```

Comment:
Could you update the documentation https://github.com/threatgrid/ctia/blob/master/doc/properties.org with with new property?

Reply:
To be more explicit, could you rename the property to something like `ctia.http.bundle.max-exported-relationships` or `ctia.http.bundle.export.max-relationships`